test/Tests/src/Semantic/Meta_Spec.enso

```diff
@@ -277,9 +277,16 @@ spec =
 
             Meta.get_annotation value "my_method" "self" . should_equal "self"
 
-            Meta.get_annotation value "Value" "foo" 7 8 . should_equal 15
```

Comment:
This test was introduced by @4e6 in #4049, but I don't think it is correct. At least according to #6612 people expect that the `self` value is going to be `My_Type` itself. Negating the test to expect `Nothing` and adding `"get annotations on constructor"` set of checks below.